### DISCUSSION TITLE
Question about transforms.Invertd (roi mask in original image)

### DISCUSSION THREAD

**Discussion by XIANYUNYEHE-DEL** [Project-MONAI/MONAI#5105] [Q&A]
Hi, if i use `transforms.CropForegroundd` like this
```
transforms.CropForegroundd(keys=["image", "label"], source_key="label"),
```
label stands for mask.(For example, using mask to get a smaller region in original image for segmentation)
How do I set the parameters in the `transforms.Invertd` in inference stage.

**Comment by wyli** [comment]
there are some examples for the transform inverse :
- https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/torch/unet_inference_dict.py
- https://github.com/Project-MONAI/tutorials/blob/main/modules/inverse_transforms_and_test_time_augmentations.ipynb
